Js/popUp.js

```diff
@@ -0,0 +1,142 @@
+    liveDemo: 'https://tarikbouari.github.io/My-Portfolio-Website/',
+    source: 'https://github.com/tarikbouari/My-Portfolio-Website',
+  },
+  {
+    id: 'project5',
+    title: 'Multi-Post stories',
+    img: 'images/dance.png',
+    subtitle: ['Canopy', 'Backend', '2015'],
+    technologies: ['javascript', 'Css', 'Html','Github','ruby','bootstrap'],
+    description:
+      'hi Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    liveDemo: 'https://tarikbouari.github.io/My-Portfolio-Website/',
+    source: 'https://github.com/tarikbouari/My-Portfolio-Website',
+  },
+];
+
+const proContainer = document.getElementById('pro-container');
+const boxContainer = document.createElement('div');
+boxContainer.className = 'note';
+const mainPage = document.getElementById('projects');
+const about = document.getElementById('about');
+const footerSection = document.querySelector('footer');
+const presentation = document.getElementById('presentation');
+const header = document.querySelector('header');
+const btnProject = document.querySelectorAll('.primary-button');
+
+function addBlur() {
+  presentation.classList.add('blur');
+  header.classList.add('blur');
+  mainPage.classList.add('blur');
+  footerSection.classList.add('blur');
+  about.classList.add('blur');
+  mainPage.style.display = 'none';
+  footerSection.style.display = 'none';
+}
+
+function removeBlur() {
+  presentation.classList.remove('blur');
+  header.classList.remove('blur');
+  mainPage.classList.remove('blur');
+  footerSection.classList.remove('blur');
+  about.classList.remove('blur');
+}
+
+function projectLoad(item) {
+  const box = `
+  <div class="title-container">
+   <h2 class='title'>${item.title}</h2>
+  <img src="images/close.svg" id="closebtn" class="closebtn" alt="close-icon">
+  </div>
+
+  <ul class="specific">
+  <li>${item.subtitle[0]}</li>
+  <li>${item.subtitle[1]}</li>
+  <li>${item.subtitle[2]}</li>
+  </ul>
+  <img class="image" src='${item.img}'>
+  <div class="box-lang">
+  <p>${item.description}</p>
+  <div class=""box-tech>
+  <ul class="language">
+  <li>${item.technologies[0]}</li>
+  <li>${item.technologies[1]}</li>
+  <li>${item.technologies[2]}</li>
+  </ul>
+  <ul class="languages">
+  <li>Github</li>
+  <li>Ruby</li>
+  <li>Bootstrap</li>
+  </ul>
+  <div class="links">
+  <a href="${item.liveDemo}"><span>See live</span>   <img src="/images/arraw.svg" alt="ellipse icon"></a>
+ 
+  <a href="${item.source}"><span>See Source</span>  <img src="/images/github.svg" alt="ellipse icon"></a>
```

Comment:
### Your buttons
![buttons-icons](https://user-images.githubusercontent.com/93039359/180114914-a577d043-ed59-4feb-b594-2c5a02c780a2.png)

- [ ] Your buttons icons are not visible because your link should contain a dot `(.)`. Please add a dot at the start of each image link to show the icons :+1: